examples/open-modal/package.json

```diff
@@ -39,6 +39,7 @@
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.0",
+    "babel-plugin-inline-json-import": "^0.2.1",
```

Comment:
delete

Reply:
We are loading a json file in the open-modal example:
__src/configurations/config.json__
it's being imported in __fresh-map.js

This example is currently broken in master branch